packages/trie/src/types.ts

```diff
@@ -8,9 +8,17 @@ export type TrieNode = BranchNode | ExtensionNode | LeafNode
 
 export type Nibbles = number[]
 
+// [ encodedPath, key ]
```

Comment:
Hmm, I find this documentation somewhat misleading. The `key` in this context would be the pointer to whatever hash this `ExtensionNode` points to. I think this is implied to by "key" but this might be confused with the MPT-keys (so the ones you use as `get` argument). Not sure what a better term here would be though. `referencedHash`? Or maybe `pointerHash`?

Reply:
I've copied what is in the ethereum.org documentation. I agree that it is not immediately obvious what this represents. I will remove these two comments and instead add a link to the docs.

Reply:
I dunno if I like the updated comment any better. The docs do explain what the extension node is, in that the `key` is the DB key for the next node. 

Can we get rid of the "raw node" terminology and just have extension nodes, which are simply the encoded path to the node and then the DB key for the next lookup, and then lead nodes, which are the encoded path and then the leaf value (which is either an rlp encoded account or null, right)?

Reply:
Yeah that's fair. 

I'm not sure I get the idea behind getting rid of the "raw node" terminology. We could do a renaming, but then we would have to rework the classes since these are already called "ExensionNode" and "LeafNode". Additionally, in comments and various places (e.g. methods like `decodeRawNode`), we have been referring to "raw nodes" quite often, and I would tend not to make changes that are too intrusive. 

I've updated the comment, let me know if that seems clear enough